paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
On 18 July 2016 at 17:32, Tianxiang Zhang <notifications@github.com> wrote:

> In paper-typeahead-input.html
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30#discussion_r71157635>
> :
>
> > @@ -313,7 +340,7 @@
> >          }
> >        });
> >        var input = Polymer.dom(this.root).querySelector('paper-input');
> > -      input.$.input.autocomplete = 'address-level4';
> > +      input.$.input.autocomplete = this.autocomplete;
>
> Got it. But if I didn't remember wrong, I have been searching for the
> chrome suggesting problem a lot before and the conclusion of discussions I
> found is that autocomplete="Off" is not reliable, workaround is to use
> autocomplete="address-level4". But it was quite ago, I am not sure if
> Chrome made some updates to change the situation here. If you like, can you
> search for some docs or evidence to say autocomplete="Off" is working now?
> Also, you change it to be "Off" is it because the original implementation
> using autocomplete="address-level4" is not working?
>
I just read a lot about this issue and I understand what you meaning. I
will try to find out why Its working for me and let you know about the
result.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30/files/dfdc34b0472906d1f9c7005d4442b0162ab8c4f8#r71157635>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG0SnMv-cJiCcUWVzZlFlEDwWauwEf8Fks5qW46DgaJpZM4JKWUL>
> .
>



-- 
C уважением, Пличко Павел